{"uuid": "670c28d8-9c30-4a4e-a04b-6801f0c0150a", "children": ["073e79eb-4114-4b77-94b0-799d88e96080", "20a0d9df-60ce-45c8-b620-629a55fe518a"], "befores": [{"name": "reset_scenario_lf", "status": "passed", "start": 1702379882511, "stop": 1702379895298}], "afters": [{"name": "reset_scenario_lf::1", "status": "passed", "start": 1702379981387, "stop": 1702379981387}, {"name": "reset_scenario_lf::teardown_session", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/usr/local/lib/python3.8/dist-packages/allure_commons/_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/imgd/ctpriv-scripts/altice/wlan-testing/tests/e2e/basic/conftest.py\", line 152, in teardown_session\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/lanforge/lf_tools.py\", line 470, in reset_scenario\n    self.Chamber_View()\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/lanforge/lf_tools.py\", line 503, in Chamber_View\n    self.CreateChamberview.clean_cv_scenario(scenario_name=self.scenario_name)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../lanforge/lanforge-scripts/py-scripts/create_chamberview.py\", line 66, in clean_cv_scenario\n    self.rm_cv_text_blob(cv_type, scenario_name)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/lanforge/lanforge-scripts/py-json/cv_test_manager.py\", line 240, in rm_cv_text_blob\n    self.json_post(req_url, data)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/lanforge/lanforge-scripts/py-json/LANforge/lfcli_base.py\", line 256, in json_post\n    json_response = lf_r.json_post(show_error=debug_,\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/lanforge/lanforge-scripts/py-json/LANforge/LFRequest.py\", line 176, in json_post\n    resp = urllib.request.urlopen(myrequest)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 222, in urlopen\n    return opener.open(url, data, timeout)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 525, in open\n    response = self._open(req, data)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 542, in _open\n    result = self._call_chain(self.handle_open, protocol, protocol +\n  File \"/usr/lib/python3.8/urllib/request.py\", line 502, in _call_chain\n    result = func(*args)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 1383, in http_open\n    return self.do_open(http.client.HTTPConnection, req)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 1354, in do_open\n    h.request(req.get_method(), req.selector, req.data, headers,\n  File \"/usr/lib/python3.8/http/client.py\", line 1256, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n  File \"/usr/lib/python3.8/http/client.py\", line 1302, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n  File \"/usr/lib/python3.8/http/client.py\", line 1251, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n  File \"/usr/lib/python3.8/http/client.py\", line 1011, in _send_output\n    self.send(msg)\n  File \"/usr/lib/python3.8/http/client.py\", line 951, in send\n    self.connect()\n  File \"/usr/lib/python3.8/http/client.py\", line 922, in connect\n    self.sock = self._create_connection(\n  File \"/usr/lib/python3.8/socket.py\", line 796, in create_connection\n    sock.connect(sa)\n"}, "start": 1702379981387, "stop": 1702379981515}], "start": 1702379882511, "stop": 1702379981516}